src/openms/source/FORMAT/FileHandler.cpp

```diff
@@ -409,6 +409,13 @@ namespace OpenMS
       }
     }
 
+    // mzTab file format
+    for (Size i = 0; i != complete_file.size(); ++i) {
+        if (complete_file[i].hasSubstring("MTD mzTab-version")) {
```

Comment:
I think separator is not correct. should be \t between MTD and mzTab

Reply:
Thanks, good catch. I fixed it.